docs/administration-guide/09-setting-up-slack.md

```diff
@@ -1,33 +1,48 @@
 
-Then just click on the large and conveniently placed button `Create a Slack Bot User for Metabot` which will open a new browser tab and send you over to Slack to create the Bot user account.
+## The app manifest
 
-Click over to the tab that was opened and you'll now be on the Slack Bot creation page.
+To get the app manifest, head back to Metabase and the Slack settings page, copy the manifest, and return to the Slack app creation page on the Slack website to paste in the manifest (the manifest is in YAML format). The manifest will take care of settings for your app and help speed things along. Once you've pasted the manifest in Slack Apps, click the **Next** button. Then hit **Create** to set up your Slack App.
 
-![Slack API Auth](images/SlackAPIAuth.png)
+## Install your app to your workspace
 
-Now give the Bot user a helpful name (we suggest `Metabot`) and click the `Add bot integration` button and a bot user will be generated for you.  Look for the Bot's API token in the next page. It will look like `xoxp-etc-etc-etc` and all you need to do is copy that value and head back to Metabase.
+On the Slack site for your newly created app, in the **Settings** > **Basic Information** tab, under **Install your app**, click on **Install to workspace**. On the next screen, click **Allow** to give Metabase access to your Slack workspace.
 
-Paste the value into the text box for `Slack API Token` and click the button to save your changes.
+## The Bot User OAuth Token
 
-Now go to Slack and create a new channel named `metabase_files`. Due to the Slack api, we'll need this to attach graphs to pulses and Metabot answers.
+On the Slack site page for your Slack app, on the left in the **Features** section, click on **OAuth and Permissions** in the Slack Apps sidebar and then copy the **Bot User OAuth Token**. Return to the Slack settings page in your Metabase and paste this token in the Metabase field with the same name.
 
-That's it!  Metabase will automatically run a quick test to check that the API token is working properly and if not you'll get an error message.
+## Create a dedicated Metabase channel in your Slack
+
+In your Slack workspace, create a public channel named whatever you want — we think something like "Metabase" does just fine — then enter that channel's name in the **Slack Channel Name** field in Metabase. This channel allows your Metabase to post to your Slack workspace without having to deal with unnecessary permissions. Due to the way the Slack API is set up, we’ll need this channel to attach charts Dashboard Subscriptions. Make sure the channel you create is the same channel that you enter in this field in Metabase. 
```

Comment:
"Metabase" (our suggested channel name) should be lowercase ("metabase") — Slack doesn't let you enter capitalized channel names.

Reply:
"This channel allows your Metabase to post to your Slack workspace without having to deal with unnecessary permissions. Due to the way the Slack API is set up, we’ll need this channel to attach charts Dashboard Subscriptions." 

Could probably be consolidated and I think there's a typo with "attach charts Dashboard Subscriptions"